test/compflags/lydia/library/exportArray/callExported2Arg.test.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "exportFuncWith1Arr1Not.h"
+
+extern void chpl_library_init(int argc, char* argv[]);
+extern void chpl_library_finalize(void);
+
+// Test of calling an exported function that takes an array
+int main(int argc, char* argv[]) {
+  // Initialize the Chapel runtime and standard modules
+  chpl_library_init(argc, argv);
+
+  // Call the function
+  int64_t x[5] = {1, 2, 3, 4, 5};
+  chpl_external_array arrX = chpl_make_external_array_ptr(x, 5);
+  foo(&arrX, 7);
```

Comment:
I'm surprised by the `&` here. I would have expected the struct to be passed directly (vs a pointer to it).